app/views/groups/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<%= render "shared/nav" %>
+
+<div class="categories">
+  <% if @groups.empty? %>
+  <div class="get-started">
+    <h3 class="started">Get Started</h3>
+    <span class="div">Add your first budget by clicking on the button below</span>
+  </div>
+  <% else %>
+
+  <div class="categories-container">
+    <% @groups.each do |group| %>
+      <div class="category-card">
+        <div class="category-icon">
+          <%= image_tag group.icon, alt: 'icon', class: "icon" %>
+        </div>
+        <div class="card-text">
+          <div class="top-card">
+            <%= link_to group_path(group.id.to_s), style: "text-decoration: none;" do %>
+              <p class="category-name"><%= group.name %></p>
+            <% end %>
+            <p class="amount">$<%= group.expenses.sum(:amount) %></p>
+          </div>
+          <div class="bottom-card">
+            <p class="date"><%= group.created_at.strftime('%d %B, %Y') %></p>
+            <p class="user">by <%= current_user.name %></p>
+          </div>
+        </div>
+      </div>
+    <% end %>
+  <% end %>
+
+  <div class="fixed-button">
+    <div class="text-center link-to">
```

Comment:
- [ ] Please make sure that this button appears at the bottom of the page like in the original design. The text in the button should have a white boundary as well.
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/230722916-39131ccb-4ed2-4611-91dd-496fda14e4dc.png)
### Required
![image](https://user-images.githubusercontent.com/45460192/230722938-465bf457-f350-4340-bb2d-783a2dd08f3d.png)